lib/content-services/src/lib/pipes/is-included.pipe.ts

```diff
@@ -0,0 +1,27 @@
+/*!
+ * @license
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+    name: 'adfIsIncluded'
+})
+export class IsIncludedPipe implements PipeTransform {
```

Comment:
Can we have something like: 
```ts
@Pipe({
    name: 'adfIsIncluded'
})
export class IsIncludedPipe<T> implements PipeTransform {
    transform(value: T, array: T[]): boolean {
        return array.includes(value);
    }
}
```

It would be more generic as it will allow to use more types than string/number. Also it will ensure that type of value is the same like type of values stored in array